modules/local/bbmap/main.nf

```diff
@@ -43,7 +44,7 @@ process BBMAP_INDEX {
         mkdir ${odir}
         cp !{reference_fasta} ${odir}/reference.fasta.gz
         cd ${odir}
-        bbmap.sh ref=reference.fasta.gz t=!{task.cpus} -Xmx60g
+        bbmap.sh ref=reference.fasta.gz t=!{task.cpus} -Xmx!{bbmap_mem}g
```

Comment:
If you're tinkering with the memory allocation here consider also doing so in the process directives (i.e. replacing `label "max"`) so that the two are consistent. Otherwise we're allocating a ton of memory that the process isn't using.

Reply:
See here for more discussion: https://github.com/naobservatory/mgs-workflow/issues/36

Reply:
Implemented that in latest push.